TKG: Unable to detect zLinux31 architecture.

While executing SVT tests on zLinux31, TKG failed to detect system.os.arch as s390.

**Error:**
System.getProperty('os.arch')=s390
17:40:14  
17:40:14  System.getProperty('java.fullversion')=8.0.7.0 - pxz3180sr7-20210609_01(SR7)
17:40:14  JRE 1.8.0 Linux s390-31-Bit 20210608_6725 (JIT enabled, AOT enabled)
17:40:14  OpenJ9   - 46b10e7
17:40:14  OMR      - e4c94e9
17:40:14  IBM      - c24a144
17:40:14  
17:40:14  Cannot determine System.getProperty('os.arch')=s390
17:40:14  
17:40:14  makefile:66: recipe for target 'envDetect' failed

Link:- hyc-runtimes-jenkins.swg-devops.com/job/Grinder/16159

--- a/src/org/openj9/envInfo/JavaInfo.java
+++ b/src/org/openj9/envInfo/JavaInfo.java
@@ -47,7 +47,7 @@ public class JavaInfo {
             spec += "_x86";
         } else if (osArch.contains("ppc") || osArch.contains("powerpc")) {
             spec += "_ppc";
-        } else if (osArch.contains("s390x")) {
+        } else if (osArch.contains("s390")) {
             spec += "_390";
         } else if (osArch.contains("aarch") || osArch.contains("arm")) {
             spec += "_arm";

